platform/darwin/src/MGLNetworkConfiguration.m

```diff
@@ -48,4 +55,53 @@ - (NSURLSessionConfiguration *)defaultSessionConfiguration {
     return sessionConfiguration;
 }
 
+- (void)startDownloadEvent:(NSString *)urlString type:(NSString *)resourceType {
+    if (urlString && ![self.events objectForKey:urlString]) {
+        [self.events setObject:@{ MGLStartTime: [NSDate date], MGLResourceType: resourceType } forKey:urlString];
```

Comment:
Should we use the upcoming `MMEDate` here @rclee?

Reply:
Going to refer to @alfwatt on this one but I don't think it should make much difference.